### DISCUSSION TITLE
Page.insertText() cannot set transparent value (alpha value)

### DISCUSSION THREAD

**Discussion by xinsheng** [pymupdf/PyMuPDF#910] [Q&A] [answered]
Hi Dear,

Does the api Page.insertText() can set alpha value when call?
Or if this api cannot set, is there other way that i can call so that i can add 20% transparent text to pdf page?

Thanks.

**Comment by JorjMcKie** [comment, marked as answer]
Please use `fill_opacity`/ `stroke_opacity` parameters for this.
Normally (= default render mode 0), `fill_opacity` is the relevant one, but if you use render mode 1 or 2 you may distinguish between both.
